app/assets/javascripts/discourse/app/lib/reviewable-items/base.js

```diff
@@ -1,10 +1,11 @@
-import GlimmerComponent from "discourse/components/glimmer";
 import I18n from "I18n";
 
-export default class UserMenuReviewableItem extends GlimmerComponent {
-  constructor() {
-    super(...arguments);
-    this.reviewable = this.args.item;
+export default class ReviewableItemBase {
+  constructor(reviewable, currentUser, siteSettings, site) {
```

Comment:
```suggestion
  constructor({ reviewable, currentUser, siteSettings, site }) {
```

I kind of prefer we have to pass in the right object. Otherwise, the order becomes confusing once there are alot more arguments.

Reply:
done!